tensorflow_addons/optimizers/tests/weight_decay_optimizers_test.py

```diff
@@ -401,13 +401,17 @@ def test_var_list_with_exclude_list_sgdw(dtype):
     )
 
 
+if tf.__version__[:3] > "2.8":
```

Comment:
Use ```from packaging.version import Version```

Reply:
thanks! I am switching to check if "optimizers.legacy" exists to keep consistency.